package.json

```diff
@@ -17,7 +17,10 @@
   "scripts": {
     "start": "grunt",
     "test": "grunt test",
-    "postinstall": "bower install --config.interactive=false"
+    "postinstall": [
+      "bower install --config.interactive=false",
+      "./node_modules/protractor/bin/webdriver-manager update"
```

Comment:
@mleanos I didn't like `grunt-protractor-webdriver` after testing it out..take a look at https://github.com/meanjs/mean/pull/939 for the addition of the webdriver update to the grunt file.